addons/point_of_sale/wizard/pos_payment.py

```diff
@@ -61,10 +61,10 @@ def check(self):
                 'payment_method_id': init_data['payment_method_id'][0],
             })
 
-        if order._is_pos_order_paid():
-            order.action_pos_order_paid()
-            order._create_order_picking()
-            order._compute_total_cost_in_real_time()
+        if order.state == 'draft' and order._is_pos_order_paid():
+            order._process_saved_order(False)
+            if order.state in {'paid', 'done', 'invoiced'}:
+                order._send_order()
```

Comment:
Should be in `pos_self_order` order._send_order() will trigger a traceback if self isn't installed

Reply:
Indeed, thanks. Moved _send_order definition to point_of_sale.